.github/workflows/build_and_push.yml

```diff
@@ -119,13 +116,14 @@ jobs:
           OPENEBS_NAMESPACE: "openebs"
         run: |
           make ci
+          make helm-install
           make sanity
 
       - name: Upload CI Test Coverage Report
         uses: codecov/codecov-action@v4
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
-          file: ./tests/bdd_coverage.txt
+          files: ./tests/bdd_coverage.txt,./tests/bdd_coverage_custom-node-id.txt
```

Comment:
Seems like you need to rebase your branch properly, I see changes from previous commits. cc @tiagolobocastro ?

Reply:
Yeah, strange, seeing some of my commits here. I'd suggest fetching the latest develop, then rebasing or even re-applying your changes on top of it.

Reply:
I rebased the brach. Could you check if its all ok?

Reply:
Seems like it, thanks you!